source/core/multikey-index-bounds.txt

```diff
@@ -349,10 +349,10 @@ field ``ratings`` to require that the array contains at least one
 
 Taking the predicates separately:
 
-- the bounds for the ``score: { $lte: 5 }`` predicate is ``[ -Infinity, 5
-  ]``;
+- the bounds for the ``score: { $lte: 5 }`` predicate are ``[ [ -Infinity, 5
```

Comment:
I agree with changing "is" to "are", but what is the motivation for the double square brackets?

Reply:
It apprears like this in all other places on this page. Search "[ [" to see the other occurances.
It makes sense to me because bounds are arrays of arrays. For example [ [1,3], [8,10] ]